typescript/sdk/src/warp/WarpCore.ts

```diff
@@ -433,6 +444,10 @@ export class WarpCore {
     }
 
     const adapter = destinationToken.getAdapter(this.multiProvider);
+    if (isXERC20Adapter(adapter)) {
+      return await adapter.belowMintLimit(originTokenAmount.amount);
```

Comment:
For consistency, it might be nice for your new method to simply return the limit itself and let the caller compare like we do already below here